doc/developer/guide.md

```diff
@@ -105,17 +105,17 @@ You will need JDK 8 or 11. The easiest way to install this is via Homebrew:
 brew install --cask homebrew/cask-versions/temurin11
 ```
 
-Then, download and extract the Confluent Platform tarball:
+Then, download and extract the Confluent Platform tarball (when using bash, replace `~/.zshrc` with `~/.bashrc`):
 
 ```shell
 INSTALL_DIR=$HOME/confluent  # You can choose somewhere else if you like.
 mkdir $INSTALL_DIR
 curl http://packages.confluent.io/archive/7.0/confluent-7.0.1.tar.gz | tar -xzC $INSTALL_DIR --strip-components=1
-echo export CONFLUENT_HOME=$(cd $INSTALL_DIR && pwd) >> ~/.bashrc
-source ~/.bashrc
+echo export CONFLUENT_HOME=$(cd $INSTALL_DIR && pwd) >> ~/.zshrc
+source ~/.zshrc
```

Comment:
Changed snippet in this macOs specific section to zsh since this is now the default shell. Added comment regarding bash before the snippet.

Reply:
I'm not sure if most people are using zsh or bash nowadays.

Reply:
Let me know if I should revert this change.

The motivation was that zsh is the default shell on macOs since 2019 (https://www.theverge.com/2019/6/4/18651872/apple-macos-catalina-zsh-bash-shell-replacement-features) and that this section / code snippet is macOs specific.